src/main/scala/cromwell/binding/values/WdlFileValue.scala

```diff
@@ -0,0 +1,25 @@
+package cromwell.binding.values
+
+import java.io.File
+import cromwell.binding.types.WdlFileType
+
+case class WdlFileValue(value: File) extends WdlPrimitiveValue {
```

Comment:
That's an interesting idea, once this all settles down that might be worth revisiting.